README.md

```diff
@@ -46,9 +46,10 @@ Algorand is an open-source, proof of stake blockchain and smart contract computi
 
 ## Wallets
 
-- [Pera Wallet](https://github.com/perawallet) - Secure, open source and community driven wallet for mobile devices. Maintained by the team behind official Algorand Wallet. 
+- [Pera Wallet](https://github.com/perawallet) - Secure, open source and community driven wallet for mobile devices. Maintained by the team behind official Algorand Wallet.
 - [MyAlgo Connect](https://github.com/randlabs/myalgo-connect) - Web Wallet and JS library for signing transactions in the browser.
 - [AlgoSigner](https://github.com/PureStake/algosigner) - Browser plugin Web Wallet and JS library for signing transactions in the browser.
+- [Method Wallet](https://methodwallet.app/) - Algorand Wallet you'll love.
```

Comment:
Thank you for contributing! 

Could you tweak the description a little bit to include brief mention of distinct features this wallet offers. In other words, instead of 'you'll love' it should mention why would someone love it :)

Reply:
@salvatorecorvaglia actually nvm, ill tweak it myself in a separate PR